core/src/themes/ios/default.tokens.ts

```diff
@@ -71,4 +79,52 @@ export const defaultTheme: DefaultTheme = {
     xxxl: 'var(--ion-radii-500)',
     xxxxl: 'var(--ion-radii-full)',
   },
+  components: {
```

Comment:
There's a lot of components that have been added due to dark, high contrast, and high contrast dark share the same variable value (light overrides it in its file) This leads to a lot of variables being created. I wonder if maybe we should consider creating component theme files instead? Like `action-sheet.dark.scss` and loading that file when dark mode is enabled.